en/docs/guides/password-mgt/recaptcha-invalid-password-attempts.md

```diff
@@ -0,0 +1,106 @@
+# ReCaptcha On Invalid Password Attempts
+
+The password account recovery feature implemented in the WSO2 Identity Server helps to recover the password of the account in case the user forgets it. This recovery process can also be secured with captcha verification.
+
+By configuring reCaptcha, you can mitigate or block brute force attacks.
+
+!!! info 
+    For more information on setting up password recovery, see [Password
+    Recovery](../../../guides/identity-lifecycles/recover-username/).
+
+    For more information on brute force attacks, see [Mitigating Brute Force
+    Attacks](../../../deploy/mitigate-attacks/mitigate-brute-force-attacks).
+
+---
+
+{!fragments/set-up-recaptcha.md!}
+
+There are two ways to configure this feature.
```

Comment:
Do we need these listed inline and linked? We stopped listing it within the page because of the right nav right?

Reply:
yeah true..I couldn't think of another way to express the idea that we have two ways to configure the feature and that the two subheadings are the two ways. Any ideas?

Reply:
Why don't we just have "There are two ways to configure this feature." and mention the topics straight away?